charon-ml/src/Types.ml

```diff
@@ -315,27 +314,15 @@ and generic_args = {
 and trait_instance_id =
```

Comment:
I think we should keep the names consistent between the Rust code and the ML code: we should rename `trait_instance_id` (shouldn't be more than a `sed` right?).

Reply:
I'm waiting for the dust to settle first: as you pointed out this may not be the best name, and I expect other naming will have to change.

Reply:
Opened https://github.com/AeneasVerif/charon/issues/287